titan/params/hiv.yml

```diff
@@ -31,6 +31,19 @@ hiv:
       type: float
       min: 0
       max: 1
+    haart_scale:
+      non_adherent:
```

Comment:
I think I saw code way back in the review that was still using the old version of this param - can you double check?